subprojects/docs/src/docs/userguide/img/jettyPluginTasks.graphml

```diff
@@ -1,101 +1,101 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:y="http://www.yworks.com/xml/graphml" xmlns:yed="http://www.yworks.com/xml/yed/3" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://www.yworks.com/xml/schema/graphml/1.1/ygraphml.xsd">

```

Comment:
Looks like the default for yEd is to use Windows line endings. Does it make sense to change it?

Reply:
It's an XML document under the hood. Does it matter?

Reply:
I'd expect if you change it with yEd it will again make it CRLF again.

Reply:
It does not. It keeps the same file endings. I just downloaded yEdit, made a change to this document and ran `file` this is the output.

```
file subprojects/docs/src/docs/userguide/img/jettyPluginTasks.graphml 
subprojects/docs/src/docs/userguide/img/jettyPluginTasks.graphml: XML 1.0 document text, ASCII text, with very long lines
```